docs/Telemetry.md

```diff
@@ -215,6 +215,10 @@ Refer to the below table to understand different critical telemetry events raise
 | `ReceivedNullOrEmptyToken` | On receiving null or empty auth token |
 | `SystemMessageReceived` | On system message received |
 | `HistoryMessageReceived` | On history message received |
+| `ExecuteSigninCardCallbackRequest` | On Signin adaptive card received from bot |
```

Comment:
Would `sign-in card` be better?

That's how the documentation call it:

https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference#sign-in-card

Reply:
done